<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationPage</class>
 <widget class="QDialog" name="RegistrationPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f7fdf9; font-size:20px; text-align: center;</string>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { border-radius: 4px;  color:rgb(0, 0, 0);  background-color: #f7fdf9; } QLineEdit:focus {  border-style:outset;border-width:4px; border-radius: 4px; border-color: #1db954; color: #f7fdf9; background-color: #181818; }</string>
   </property>
   <property name="placeholderText">
    <string>логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>121</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { border-radius: 4px;  color:rgb(0, 0, 0);  background-color: #f7fdf9; } QLineEdit:focus {  border-style:outset;border-width:4px; border-radius: 4px; border-color: #1db954; color: #f7fdf9; background-color: #181818; }</string>
   </property>
   <property name="placeholderText">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>161</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { border-radius: 4px;  color:rgb(0, 0, 0);  background-color: #f7fdf9; } QLineEdit:focus {  border-style:outset;border-width:4px; border-radius: 4px; border-color: #1db954; color: #f7fdf9; background-color: #181818; }</string>
   </property>
   <property name="placeholderText">
    <string>пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_repeat_line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>201</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { border-radius: 4px;  color:rgb(0, 0, 0);  background-color: #f7fdf9; } QLineEdit:focus {  border-style:outset;border-width:4px; border-radius: 4px; border-color: #1db954; color: #f7fdf9; background-color: #181818; }</string>
   </property>
   <property name="placeholderText">
    <string>пароль (repeat)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: #1db954; color: #f7fdf9; border-radius: 20px; position: relative;} QPushButton:focus {position: absolute; left: -5%; width: 110%; top: -5%; height: 110%;}</string>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>240</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="status_label">
      <property name="styleSheet">
       <string notr="true">color: #f7fdf9; font-size:20px; text-align: center;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
